docs/CONTRIBUTING.md

```diff
@@ -73,36 +72,56 @@ A component is the basic frontend building block of a budibase app.
 
 ### Component Library
 
-Component libraries are collections of components as well as the definition of their props contained in a file called `components.json`. 
+Component libraries are collections of components as well as the definition of their props contained in a file called `components.json`.
 
 ## Contributing to Budibase
 
-* Please maintain the existing code style. 
+- Please maintain the existing code style.
 
-* Please try to keep your commits small and focused.
+- Please try to keep your commits small and focused.
 
-* Please write tests.
+- Please write tests.
 
-* If the project diverges from your branch, please rebase instead of merging. This makes the commit graph easier to read.
+- If the project diverges from your branch, please rebase instead of merging. This makes the commit graph easier to read.
 
-* Once your work is completed, please raise a PR against the `develop` branch with some information about what has changed and why.
+- Once your work is completed, please raise a PR against the `develop` branch with some information about what has changed and why.
 
 ### Getting Started For Contributors
-#### 1.  Prerequisites
 
-NodeJS Version `14.x.x`
+#### 1. Prerequisites
+
+- NodeJS version `14.x.x`
+- Python version `3.x`
+
+#### 1.1 Using asdf (recommended)
+
+Asdf is a package manager that allows managing multiple dependencies.
+
+1. Install using script (recommended):
```

Comment:
Very minor: the `1.` and `2.` implies to me this is a set of instructions to follow. So reads like first install using this script then do these manual steps. Might be clearer as two bullets instead

Reply:
Good shout